### ISSUE TITLE
Inconsistend results in the course of 24 hours

### ISSUE THREAD

**Issue by coralexbadea** [CorentinJ/Real-Time-Voice-Cloning#588]
I run the program with great success, I was really impresses. The next day the program was run with the same audioinput and the same sentence, but the output was noisy and not good at all. Any idea why this might happen?

**Comment by blue-fish** [comment]
It's most likely due to the configuration of your system changing over that time.

However, it could also be one of these two things:
1. Dropout. The synthesizer (tacotron) has a prenet in the encoder and decoder where some elements are randomly set to zero. This is mainly done to prevent model overfitting during training, but is also used in inference to get some variation in the output. If you click the "random seed" button on the UI, it will make generation deterministic.
2. The file used to generate the speaker embedding must be identical. Any variation may lead to a different embedding and the synthesizer output is sometimes sensitive to small differences in the embedding.

If you study this issue further and have evidence of a bug, please submit it and we'll troubleshoot it further. However, I spent a lot of time on deterministic generation (#432) and think it's unlikely there is a bug (at least not one that affects all users).
